src/modelbench/hazards.py

```diff
@@ -154,62 +169,15 @@ def score(self, sut_scores: Mapping[str, TestRecord]) -> "HazardScore":
             exceptions=num_exceptions,
         )
 
-    hazard_key = None
-
     def tests(self, secrets: RawSecrets) -> List[BaseTest]:
-        if self.hazard_key is None:
-            raise (NotImplementedError("must set hazard key"))
         if not self._tests:
-            self._tests = [TESTS.make_instance(f"safe-{self.hazard_key}-1.0", secrets=secrets)]
+            self._tests = [TESTS.make_instance(f"safe-{self.hazard_key}-{self.locale}-1.0", secrets=secrets)]
         return self._tests
 
+    def _uid(self):
```

Comment:
And also on line 177. One of the goals of HasUid is to avoid hand-written string composition as is done on 177. So perhaps this could be something like {"class":"self", "locale": self.locale, "version": "1.0"}

Reply:
Okay updated. It's a little inconvenient to have to create a method for each individual instance attribute in the uid, but it's not a huge deal.